en/user-guide/layerexplanations/layerexplanation.rst

```diff
@@ -27,33 +32,33 @@ Main layer for line-wastewater structures (channels). Creating a new record in t
 Wastewater Structures
 ---------------------
 
-od_wastewater_structure
-^^^^^^^^^^^^^^^^^^^^^^^
+Wastewater Structures (od_wastewater_structure)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 This layer shows the detailed geometries of wastewater structures. You digitize a new detailed geometry by the records of the layer vw_qgep_wastewater_structure. If you want to change an existing geometry, then you have to edit in the layer od_wastewater_structure. In database-schema qgep_od.
 
-Structure Parts
-^^^^^^^^^^^^^^^
+Structure Parts (od_structure_part)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Cover is the only structure part with a point-geometry himself. All other structure parts are just linked to there wastewater structures and should by only edited by the main layers (vw_qgep_wastewater_structure and vw_qgep_reach).
 
-vw_cover
-^^^^^^^^
+Covers (vw_cover)
+^^^^^^^^^^^^^^^^^
 
 You use this layer, if you want to change the situation of just one selected cover (and not the hole wastewater structure) or if you want to add a new cover to an existing wastewater structure. (you can add an additional cover also in the covers-tab of the vw_qgep_wastewater_structure, but this cover has no situation...). And you use this layer if you want to show the detail position of the covers (e.g. in network_plan or pipeline_registry) or to export the cover positions. In database-schema qgep_od.
```

Comment:
I suggest to use the name of the geometry attribut: situation_geometry . And also rephrase it here: And you use this layer if you want to show the detail position of the covers (e.g. in network_plan or pipeline_registry) or to export the cover positions. -> And you use this layer if you want to show the detailed situation_geometry of the covers (e.g. in network_plan or pipeline_registry) or to export the cover situation_geometry.

Reply:
...but this cover has no situation...   This is no longer true - with QGIS 3.16 you can digitize a Childobject with  situation.

Reply:
Fixed (removed the part where it is mentionned that we create covers with no geometry)